data/journeys/journeys.md

```diff
@@ -27,4 +27,49 @@ Milestone: Scale
   Major action
   Major action
   ...
-```
\ No newline at end of file
+```
+
+## New Ansible user
```

Comment:
How do you feel about "Novice" instead? Do you think that has a negative connotation, almost like "newbie"? Consider someone who is a solutions architect maybe or expert in some other field, would being labelled a novice seem almost condescending?

Reply:
We used to worry about the same thing when I lead gaming groups, "newbie" can be demeaning. Perhaps "New to Ansible" covers all bases?

Reply:
that makes some sense. perhaps we can add a line underneath to explain what we mean by novice instead. happy to do that if you like the idea

Reply:
I could also go with "Seeker" because I play too many games :stuck_out_tongue:

Reply:
adding a line underneath sounds like a good idea